member/src/main/proto/com/improving/app/member/domain/improving-app-member-domain.proto

```diff
@@ -0,0 +1,200 @@
+syntax =  "proto3";
+
+package com.improving.app.member.domain;
+
+import "com/improving/app/organization/domain/improving-app-organization-domain.proto";
+import "com/improving/app/tenant/domain/improving-app-tenant-domain.proto";
+import "google/protobuf/timestamp.proto";
+import "scalapb/scalapb.proto";
+
+option (scalapb.options) = {
+  no_default_values_in_constructor: true
+  preserve_unknown_fields: false
+};
+
+
+//Data Model
+message MemberId {
```

Comment:
Please put the MemberId message into the common project as it will be widely used.

Reply:
will do

Reply:
done in @miguelTaningcoYoppworks PR